src/main/java/com/google/api/codegen/viewmodel/metadata/VersionIndexView.java

```diff
@@ -51,6 +51,9 @@ public String resourceRoot() {
 
   public abstract List<VersionIndexRequireView> requireViews();
 
+  @Nullable
+  public abstract List<String> requireTypes();
```

Comment:
Should this be merged with the existing `requireViews` attribute?

Reply:
I have no idea. There wasn't much to work with from the comments so I thought it best to keep it standalone.

Reply:
I see. In that case, I would support merging them. (I would expect a `VersionIndexRequireView` to be a ViewModel for representing a Ruby `require` statement in the Version Index, so it would be surprising to need two separate lists of these.)

Reply:
Actually I take that back. That RequireView model thing has properties that are not really relevant to this type of require statement and would be confusing (clientName, etc.)

I think we'd want a new view model or need to split into a two types. Since there's only one property right now I'd propose we just leave it as a string and promote it to it's own type if, and when, we need others (and I can't think of any).